test/lib/rules/no-disallowed-lwc-imports.js

```diff
@@ -0,0 +1,266 @@
+                ),
+            },
+        ],
+    },
+    {
+        code: `export { ShouldNotImport, LightningElement } from "lwc"`,
+        errors: [
+            {
+                message: new RegExp(
+                    `Invalid export. "ShouldNotImport" is not a known and stable API.`,
+                ),
+            },
+        ],
+    },
+    {
+        code: `export { LightningElement, ShouldNotImport } from "lwc"`,
+        errors: [
+            {
+                message: new RegExp(
+                    `Invalid export. "ShouldNotImport" is not a known and stable API.`,
+                ),
+            },
+        ],
+    },
+    {
+        code: `export { ShouldNotImport, AlsoBanned } from "lwc"`,
+        errors: [
+            {
+                message: new RegExp(
+                    `Invalid export. "ShouldNotImport" is not a known and stable API.`,
+                ),
+            },
+            {
+                message: new RegExp(`Invalid export. "AlsoBanned" is not a known and stable API.`),
+            },
+        ],
+    },
+    {
+        code: `export { LightningElement, ShouldNotImport, AlsoBanned } from "lwc"`,
+        errors: [
+            {
+                message: new RegExp(
+                    `Invalid export. "ShouldNotImport" is not a known and stable API.`,
+                ),
+            },
+            {
+                message: new RegExp(`Invalid export. "AlsoBanned" is not a known and stable API.`),
+            },
+        ],
+    },
+    {
+        code: `export {} from "lwc"`,
+        errors: [
+            {
+                message: new RegExp(
+                    `Invalid export. Bare exports are not allowed on "lwc". Instead, use named exports: "export { LightningElement } from 'lwc'".`,
+                ),
+            },
+        ],
+    },
+];
+
+const validCases = [
+    {
+        code: `import { LightningElement } from "lwc"`,
+    },
+    {
+        code: `import { LightningElement, wire } from "lwc"`,
+    },
+    {
+        code: `import { LightningElement, wire, api } from "lwc"`,
+    },
+    {
+        code: `import { LightningElement as Yolo } from "lwc"`,
+    },
+    {
+        code: `import "some-other-package"`,
+    },
+    {
+        code: `import * as foo from "some-other-package"`,
+    },
+    {
+        code: `import { foo } from "some-other-package"`,
+    },
+    {
+        code: `export * as foo from "some-other-package"`,
+    },
+    {
+        code: `export * from "some-other-package"`,
+    },
+    {
+        code: `export { foo } from "some-other-package"`,
+    },
+    {
+        code: `export { LightningElement } from "lwc"`,
+    },
+    {
+        code: `export { LightningElement, wire } from "lwc"`,
+    },
+    {
+        code: `export { LightningElement, wire, api } from "lwc"`,
+    },
+    {
+        code: `export { LightningElement as Yolo } from "lwc"`,
+    },
+    {
+        code: `export {} from "some-other-package"`,
+    },
+];
```

Comment:
Add a testcase for `export { LightningElement as default } from "lwc";`

Reply:
Done!